Firmware/fw_ver_17/firmware_update_17.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
```

Comment:
Wow. I just saw the PR. Soo much code in here 😮 I got scared, initially.
Re-reading the comments in here, it seems like there are just 3 important files that changed.

I'll test them *thoroughly* and when this is a go, I'll add a review.